test/Nancy.Tests/Unit/DynamicDictionaryValueFixture.cs

```diff
@@ -987,6 +987,38 @@ public void Should_return_default_value_if_implicit_convert_fails_on_datetime()
         }
 
         [Fact]
+        public void Should_adjust_to_universal_time_when_globalizationconfiguration_datetimestyles_requires_it()
+        {
+            // Given
+            var expected = DateTime.Parse("2016-05-24T08:41:37.1661565");
+
+            var config = new GlobalizationConfiguration(new [] {"en-US"}, dateTimeStyles: DateTimeStyles.AdjustToUniversal);
+            var value = new DynamicDictionaryValue("2016-05-24T10:41:37.1661565+02:00", config);

```

Comment:
If the test is being run in a time zone which is UTC+2 *on May 24th 2016* then it won't (shouldn't) matter if it's some other offset when the test is run. If the date were in the future, you would be at the mercy of governments changing time zone rules - but it's a bit too late for them to change the rules for May 24th 2016.

The call to `DateTime.Parse` still bothers me though - I have to look up the documentation to know *exactly* what you're expecting.